lib/stack_master/parameter_resolvers/stack_output.rb

```diff
@@ -10,14 +10,14 @@ def initialize(config, stack_definition)
         @config = config
         @stack_definition = stack_definition
         @stacks = {}
+        @output_regex = %r{(?:([^:]+):)?([^:/]+)/(.+)}
```

Comment:
I'm unsure on how I feel about using a regex here. Previously this was quite simple and would just split the string using `/`.

If we go down the route of using a regex, I'd like to see this use named captures so that we can call them using a recognisable string instead of the numeric indexes. 

@stevehodgkiss @patrobinson what are your thoughts on the syntax we should expect to use here?